CMakeLists.txt

```diff
@@ -59,18 +59,10 @@ link_directories(${GSL_LIBRARY_DIRS})
 find_package(Eigen3 3.3.7 REQUIRED NO_MODULE)
 include_directories(${EIGEN3_INCLUDE_DIR})
 
-# Find Pinocchio library
 find_package(pinocchio REQUIRED)
 include_directories(${PINOCCHIO_INCLUDE_DIRS})
 link_directories(${PINOCCHIO_LIBRARY_DIRS})
 
-# Find ompl library
```

Comment:
I would put ompl back for now.
Did you successfully install this? You just need to rebuild the container.